### PR TITLE
CUMULUS-3265: Fix getGranulesForPayload function to query es

### PR DESCRIPTION
**Pull request by jennyhliu**
**Summary:** Summary of changes

Addresses [CUMULUS-3265: Bulk Delete not working from Cumulus dashboard](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3265)

## Changes

* Detailed list or prose of changes
* ...

## PR Checklist

- [x] Update CHANGELOG
- [x] Unit tests
- [x] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

**Comment by etcart** [comment]
changes look very straightforward, but I'ms truggling to see how the described issue of IDs getting turned into nulls during processing is addressed by that change, or how it manifests as a truthiness confusion in getGranulesForPayload. can you give a little explanation of how this works?

**Review by etcart — APPROVED** [review]
looks good, talked through to clarify and it all makes sense

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+### Notable Changes
+
+- The async_operation_image property of cumulus module should be updated to pull
```
Comment (Jkovarik) [inline CHANGELOG.md:11]: Is this a change in *this* PR?  Should we add it to the CHANGELOG somewhere else?     If it is, should it have this ticket as a header? @jennyhliu

Comment (jennyhliu) [reply CHANGELOG.md:11]: No. Just move the section up

**Pull request merged**